tests/client/TestPackageJson.js

```diff
@@ -0,0 +1,9 @@
+import packageJson from 'package';
```

Comment:
I don't think we should rely on using the root directory as a module path because there are a lot of folders and they could accidentally overshadow something in `node_modules`. In fact, I thought I fixed this in https://github.com/mozilla/addons-frontend/pull/2404 but I guess not, heh.

Instead, I think you should do:
````js
const packageJson = JSON.parse(readFileSync(path.join(config.get('basePath'), 'package.json')));
````

btw, this is a great test. Good idea!